magma/wire_container.py

```diff
@@ -66,7 +73,8 @@ def connect(self, other, debug_info):
         if self._driver is not None:
             old_driver = self._driver
             self.unwire(old_driver)
-            _logger.warning(
+            _logger.log(
+                getattr(logging, config.rewire_log_level),
```

Comment:
```suggestion
            _logger.log(
                py_logging.getLevelName(config.rewire_log_level),
```

See https://docs.python.org/3.10/library/logging.html#logging.getLevelName. Converts string <-> int (both directions).